File: java/latihanempat/src/main/java/com/android/cnd/latihanempat/BalokActivity.java
package com.android.cnd.latihanempat;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class BalokActivity extends AppCompatActivity {
    String title = "";
    TextView titleActivity;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_balok);
        title = getIntent().getStringExtra("KEY_TITLE");

        titleActivity = findViewById(R.id.title_balok);
        titleActivity.setText(title);
    }
}

File: kotlin/latihanempat/src/main/java/com/android/cnd/latihanempat/BalokActivity.kt
package com.android.cnd.latihanempat

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class BalokActivity : AppCompatActivity() {
    private lateinit var titleActivity: TextView
    private var title: String? = ""

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_balok)
        title = intent.getStringExtra("KEY_TITLE")
        titleActivity = findViewById(R.id.title_balok)
        titleActivity.text = title
    }
}